### ISSUE TITLE
Clarification regarding some parameters in the plots

### ISSUE THREAD

**Issue by Heimine** [ganguli-lab/degrees-of-freedom#1]
Hi, 

Thank you for the great work!
While I appreciate the insightful results in your paper, there are a few things I want to clarify in the paper:
(1) What is the epsilon value set in the figures (I notice the delta values are specified but not the epsilon values)? 
(2) When you say "base accuracy" in for example Fig. 2, does this mean the training acc or test?
(3) About the t value, when you say t refers to training steps, does it mean one epoch = one step or one batch = one step?

Thanks in advance for your time and clarification!

**Comment by bwlarsen** [comment]
(1) We plot the full curve for the threshold training dimension so that d* can be read off for any value of epsilon.  If you specify a given value of epsilon (e.g. an accuracy of 0.75) then you read off the corresponding d* as the point where the curve hits this accuracy.
(2) In Figure 2, it is the train accuracy, but we show both the train and test accuracy in the subsequent figures (Figs 3 and 6).  Note that the baseline accuracy is what we get by training the full model for the same number of epochs.
(3) One batch = one step.

Thanks for the questions!

**Comment by Heimine** [comment]
Thanks a lot!
